frontend/src/metabase/components/NumericInput.styled.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled";
+
+import InputBlurChange from "./InputBlurChange";
+
+export const NumbericImputBlurChange = styled(InputBlurChange)`
+  ${InputBlurChange.Field} {
```

Comment:
There is `fullWidth` prop available on `Input`. Would it be enough?

Reply:
The problem is that fullWidth applies `width: 100%` to both the `root` and the `field`, making it hard difficult to layout/size without another container that is sized specifically.

Reply:
Talked with @ranquild. Going to try setting on the usingInput fullWidth by default, and styling the component as needed across the app